subsys/bluetooth/mesh/pb_adv.c

```diff
@@ -926,11 +923,9 @@ static void prov_link_close(enum prov_bearer_link_status status)
 	 * be restransmitted at least three times. Retransmit the LINK_CLOSE
 	 * message until CLOSING_TIMEOUT has elapsed.
```

Comment:
You moved `CLOSING_TIMEOUT` usage to another place but left a comment nonrelavant. Please keep the code clean for better maintainability.

Reply:
Sry, my bad 👍